.github/workflows/scheduled_test_cpu.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [main, test-me-*]
   schedule:
     - cron: "0 4 * * *"
+  workflow_call:
+  pull_request:
```

Comment:
UNDO this one -- it's only to trigger the CI and check if it fixes the related tests